### DISCUSSION TITLE
Pass audio files directly to LLM instead of passing a transcript

### DISCUSSION THREAD

**Discussion by Simon-Stone** [open-webui/open-webui#7893] [Ideas]
**Is your feature request related to a problem? Please describe.**
With multimodal LLMs that can process audio now widely available, it would be very helpful to pass an audio file directly to the model, similar to how images can now be sent as part of a message.

**Describe the solution you'd like**
When uploading an audio file, a dialog popup should ask whether the file should be transcribed to text or passed as context directly.

**Comment by Simon-Stone** [comment]
Another way to implement this is to add an option in the Model settings to support Audio, similar to how Vision is handled. If the model supports Audio, uploaded files are not transcribed and passed as blobs, if it doesn't support Audio, the files are transcribed first.

**Comment by AlexM4H** [comment]
Alternatively, the option of transferring a real-time audio stream to the model

**Comment by salehardec** [comment]
@Simon-Stone  Good idea.

**Comment by CuriousLocky** [comment]
This feature would be very nice, especially when needing the model to do extra processing/extract specific information from the audio file. Always having to go through the internal STT is painful.
